extensions/amp-lightbox/1.0/amp-lightbox.css

```diff
@@ -35,3 +35,8 @@ amp-lightbox::part(lightbox) {
   background: inherit;
   color: inherit;
 }
+
+/* Prevent scrolling while lightbox is open */
+amp-lightbox[open]:root {
+  overflow: hidden;
```

Comment:
This is exactly a thing that I hope is not needed with `overscroll-behavior`

Reply:
You're right it's not needed, removed.